lib/cli_common/cli_common/utils.py

```diff
@@ -19,23 +19,22 @@ def wait_until(operation, timeout=30, interval=30):
 
 
 def retry(operation, retries=5, wait_between_retries=30):
```

Comment:
The behavior of this function has been changed to throw an exception when all the retries fail.